lambda-notes/src/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import { BrowserRouter as Router } from "react-router-dom";
```

Comment:
You are importing from `react-router-dom` but it was never added to the package.json so app will throw an error.